packages/mui-material/src/Drawer/Drawer.test.js

```diff
@@ -22,6 +30,18 @@ describe('<Drawer />', () => {
       testVariantProps: { variant: 'persistent' },
       testDeepOverrides: { slotName: 'paper', slotClassName: classes.paper },
       refInstanceof: window.HTMLDivElement,
+      slots: {
+        root: {
+          expectedClassName: classes.root,
+          testWithComponent: null,
+          testWithElement: null,
+        },
+        paper: {
```

Comment:
tests for docked, , backdrop, transition slots are missing

Reply:
Added.